harvester_e2e_tests/integration/test_hosts.py

```diff
@@ -9,8 +10,12 @@
 
 
 @pytest.fixture(scope="session")
-def focal_image_url():
-    return "http://cloud-images.ubuntu.com/releases/focal/release/ubuntu-20.04-server-cloudimg-amd64-disk-kvm.img"  # noqa
+def focal_image_url(request):
+    base_url = request.config.getoption(
+        '--image-cache-url',
+        'http://cloud-images.ubuntu.com/releases/focal/release'
+        )
+    return urljoin(base_url, "openSUSE-Tumbleweed-NET-x86_64-Current.iso")  # noqa
```

Comment:
I think this will not work in some case?
Ubuntu will not contains SUSE's tumbleweed.

And as this fixture is targeting for `focal_image_url`, the return value should not be other images.

Reply:
Oops, that's a typo, will fix it. thx!